acceptance/framework/helpers/helpers.go

```diff
@@ -358,3 +360,48 @@ func createCmdArgs(options *k8s.KubectlOptions) []string {
 	}
 	return cmdArgs
 }
+
+const DEFAULT_PAUSE_PORT = ":8517"
+
+// WaitForInput starts a http server on a random port (which is output in the logs) and waits until you
+// issue a request to that endpoint to continue the tests. This is useful for debugging tests that require
+// inspecting the current state of a running cluster and you don't need to use long sleeps.
+func WaitForInput(t *testing.T) {
+	t.Helper()
+
+	envPort := os.Getenv("CONSUL_K8S_TEST_PAUSE_PORT")
+	listenerPort := DEFAULT_PAUSE_PORT
+
+	if envPort != "" {
+		listenerPort = ":" + envPort
+	}
+
+	mux := http.NewServeMux()
+	srv := &http.Server{
+		Addr:    listenerPort,
```

Comment:
~ Maybe make the default port a number/numeric string like the override for consistency? (could be referenced as a number somewhere else later)
```suggestion
	listenerPort := os.Getenv("CONSUL_K8S_TEST_PAUSE_PORT")
	if listenerPort == "" {
		listenerPort = DEFAULT_PAUSE_PORT
	}

	mux := http.NewServeMux()
	srv := &http.Server{
		Addr:    ":" + listenerPort,
```